src/Notifications/SalesFileUploadNotification.php

```diff
@@ -37,11 +40,20 @@ public function toMail(object $notifiable)
     {
         return (new MailMessage)
             ->subject('IOI City Mall Sales File Upload Notification')
-            ->markdown('ioi-city-mall-sales-file::mail.file-upload', ['status' => $this->status, 'messages' => $this->messages]);
+            ->markdown('ioi-city-mall-sales-file::mail.file-upload', [
+                'status' => $this->status,
+                'messages' => $this->messages,
+                'receiverName' => $this->receiverName,
+            ]);
     }
 
     public function getStatus()
     {
         return $this->status;
     }
+
+    public function getReceiverName()
+    {
+        return $this->receiverName;
+    }
```

Comment:
This is for?

Reply:
Being used in `TestCase` @ `assertNotificationDetails` for verifying the receiver name

Reply:
![image](https://github.com/Retail-Cosmos/ioi-city-mall-sales-file/assets/11887260/3e4d360f-d0eb-4208-a06d-e03af9f5abed)
???